src/routes/transactions/mappers/common/transaction-info.mapper.ts

```diff
@@ -322,9 +322,16 @@ export class MultisigTransactionInfoMapper {
     }
 
     try {
+      const tx = transaction as MultisigTransaction;
```

Comment:
Are we sure this cannot be a `ModuleTransaction`? We could use the validation schema instead of casting it, for example.